unit-tests/src/test/java/org/eclipse/collections/impl/bimap/mutable/HashBiMapTest.java

```diff
@@ -102,4 +105,23 @@ public void withKeysValues()
         Assert.assertSame(map, map44);
         Assert.assertSame(map3, map4);
     }
+
+    @Test
+    public void forcePut_inverseKeyPreservation()
+    {
+        Key key1 = new Key("1");
+        Key key2 = new Key("2");
+
+        Key value1 = new Key("abc");
+        Key value2 = new Key("xyz");
+
```

Comment:
I wanted to have "equal" values in the BiMap and in the forcePut so that containsKey and containsValue are both true. So I reused value2 in the forcePut call. I could inline value2 if it is confusing.